cmd/markdown/atmos_terraform_plan_usage.md

```diff
@@ -0,0 +1,8 @@
+Examples:
+
+  – Execute a terraform plan
+
+    $ atmos terraform plan <component-name> -s <stack-name>
+
+For more information, refer to the **docs**:
+https://atmos.tools/cli/commands/terraform/usage
```

Comment:
```suggestion
For more information, refer to the [docs](https://atmos.tools/cli/commands/terraform/usage)
```

Reply:
Use markdown syntax.

Reply:
But that would ... not be relevant right? 
Terminal does not support hyperlinks on text. Gnome does it but most of them have plain text.

Reply:
Anyway did that

Reply:
The glamour engine supports formatting markdown links. It’s all about how they are styled. Also, modern terminals do let you click on links.